django_backblaze_b2/cache_account_info.py

```diff
@@ -186,6 +186,12 @@ def remove_bucket_name(self, bucket_name):
         self.cache.set("bucket_names", [n for n in self.cache.get("bucket_names", []) if n != bucket_name])
         self.cache.delete(_bucket_cachekey(bucket_name))
 
+    def list_bucket_names_ids(self) -> List[Tuple[str, str]]:
+        res = []
+        for name in self.cache.get("bucket_names"):
+            res.append((name, self.cache.get(_bucket_cachekey(name))))
+        return res
```

Comment:
unlikely, but we should handle the null case here.

Also a nit, but maybe a slightly clearer name than `res`, e.g.

```
    def list_bucket_names_ids(self) -> List[Tuple[str, str]]:
        tuples = []
        for bucket_name in self.cache.get("bucket_names", []):
            bucket_id = self.cache.get(_bucket_cachekey(bucket_name))
            tuples.append((bucket_name, bucket_id))
        return tuples
```

I can make the change if you don't get to it.

Reply:
Thanks for the feedback, I updated the PR